carddrawer/src/test/java/com/meli/android/carddrawer/model/CardAnimatorTest.kt

```diff
@@ -0,0 +1,175 @@
+        cardAnimator.switchView(FieldPosition.POSITION_FRONT)
+        Assert.assertEquals(showingLayout, FieldPosition.POSITION_FRONT)
+    }
+
+    @Test
+    fun `when call function switchView with 3 per parameter then fill showingLayout with POSITION_FRONT`() {
+        initialize()
+        val showingLayout = cardAnimator.getDeclaredField("showingLayout")
+        cardAnimator.switchView(3)
+        Assert.assertEquals(showingLayout, FieldPosition.POSITION_FRONT)
+    }
+
+    @Test
+    fun `when call function switchView with POSITION_BACK per parameter then fill showingLayout with the same value`() {
+        val mock = mockk<CardAnimator>(relaxed = true)
+        every { mock.switchView(FieldPosition.POSITION_BACK) } answers { callOriginal() }
+        mock.switchView(FieldPosition.POSITION_BACK)
+        val showingLayout = mock.getDeclaredField("showingLayout")
+        Assert.assertEquals(showingLayout, FieldPosition.POSITION_BACK)
+    }
+
+    @Test
+    fun `when call function switchViewWithoutAnimation with POSITION_FRONT per parameter fill showingLayout with POSITION_FRONT`() {
+        initialize()
+        cardAnimator.switchViewWithoutAnimation(FieldPosition.POSITION_FRONT)
+        val showingLayout = cardAnimator.getDeclaredField("showingLayout")
+        Assert.assertEquals(showingLayout, FieldPosition.POSITION_FRONT)
+    }
+
+    @Test
+    fun `when call function switchViewWithoutAnimation with POSITION_BACK per parameter fill showingLayout with POSITION_FRONT`() {
+        initialize()
+        cardAnimator.switchViewWithoutAnimation(FieldPosition.POSITION_BACK)
+        val showingLayout = cardAnimator.getDeclaredField("showingLayout")
+        Assert.assertEquals(showingLayout, FieldPosition.POSITION_BACK)
+    }
+
+    @Test
+    fun `when call function switchViewWithoutAnimation with 3 per parameter fill showingLayout with 3`() {
+        initialize()
+        cardAnimator.switchViewWithoutAnimation(3)
+        val showingLayout = cardAnimator.getDeclaredField("showingLayout")
+        Assert.assertEquals(showingLayout, 3)
+    }
+
+    @Test
+    fun `when call function saveState then showingLayout in bundle`() {
+        initialize()
+        val bundle = mockk<Bundle>(relaxed = true)
+        cardAnimator.saveState(bundle)
+        val showingLayout: Int = cardAnimator.getDeclaredField("showingLayout") as Int
+        verify(exactly = 1) {
+            bundle.putInt("showing_side", showingLayout)
+        }
+    }
+
+    @Test
+    fun `when call function saveState then showingColor in bundle`() {
+        initialize()
+        val bundle = mockk<Bundle>(relaxed = true)
+        cardAnimator.saveState(bundle)
+        val showingColor: Int = cardAnimator.getDeclaredField("showingColor") as Int
+        verify(exactly = 1) {
+            bundle.putInt("showing_color", showingColor)
+        }
+    }
+
+    @Test
+    fun `when call function showFont then call flipCardFront`() {
```

Comment:
In test description, i think that is showFront instead showFont, right?

Reply:
adjusted